CHANGES

```diff
@@ -27,7 +27,8 @@ Version 8.0.0 - In development
       CoordBBox.
     - Added tools::activeTiles that returns a vector containing all the active
       tiles in a tree that intersects a given CoordBBox.
-
+    - Added Tree::activeTileNodeCount, which counts the number and type of
```

Comment:
I really don't like this name change! This is nothing called a "TileNode", only a Tile and a Node and to make matters worse they are mutually exclusive so this new makes me nauseous :) If this method still returns the total number of active tiles is could remain "activeTileCount".

Reply:
This method cannot remain `activeTileCount()` because there's already a method called `Tree::activeTileCount()` which returns the total number of active tiles in the whole tree. This method differs in that it returns the number of active tiles per-level of the tree as a vector (similar to `Tree::nodeCount`). You obviously cannot have two methods with the same name returning something different and we cannot change the return type of the existing method without breaking API with no warning.